### PR TITLE
Update to Rust 1.15.0-nightly (ac635aa95 2016-11-18)

### PR DESCRIPTION
**Pull request by nox**

### CODE REVIEW

**Comment by nox** [comment]
Am scared: https://github.com/rust-lang/rust/issues/37786

File: derive/lib.rs [hunk 1]
```diff
@@ -66,8 +64,6 @@ fn expand_string(input: &str) -> String {
     }
 
     let tokens = quote! {
-        #type_
```
Comment (Manishearth) [inline derive/lib.rs]: Is it a decorator now?

Comment (nox) [inline derive/lib.rs]: Yes.

**Comment by Manishearth** [comment]
r=me if stuff works

**Comment by nox** [comment]
https://github.com/rust-lang/rust/issues/37788

**Pull request merged**